mergeSortedArray.java

```diff
@@ -1,3 +1,10 @@
+// Time Complexity : O(n)
```

Comment:
There are 2 arrays and so be specific when you mention O(n)

Reply:
Yeap got it! Sorry thanks